src/App.jsx

```diff
@@ -13,7 +14,8 @@ export default function App() {
         <Routes>
           <Route path="/" element={<Home />}/>
           <Route path="/NewInvoice" element={<NewInvoicePage />}/>
-        </Routes>
+         <Route path="/viewinvoice/:invoiceid" element={<ViewInvoice />}></Route>
```

Comment:
The indentation here doesn't look quite right. Do a right click -> format document